Classes/common/touchdb/TDRemoteRequest.m

```diff
@@ -174,80 +183,22 @@ - (BOOL)retry
     // request might have succeeded on the remote server, and by retrying we'd be issuing it again.
     // PUT and POST requests aren't generally idempotent, but the ones sent by the replicator are.
 
-    if (_retryCount >= kMaxRetries) return NO;
-    NSTimeInterval delay = RetryDelay(_retryCount);
-    ++_retryCount;
-    CDTLogVerbose(CDTTD_REMOTE_REQUEST_CONTEXT, @"%@: Will retry in %g sec", self, delay);
-    [self startAfterDelay:delay];
-    return YES;
+//    if (_retryCount >= kMaxRetries) return NO;
```

Comment:
It should work, but I commented it out rather than delete it to make it easier to go back if I broke something, and tbh with all the changes to try and get this working I forgot I had it commented out.